t/t1091-sparse-checkout-builtin.sh

```diff
@@ -701,6 +701,7 @@ test_expect_success 'pattern-checks: contained glob characters' '
 
 test_expect_success BSLASHPSPEC 'pattern-checks: escaped characters' '
 	git clone repo escaped &&
+	git -C escaped config index.sparse false &&
```

Comment:
Elsewhere in this patch, `advice.sparseIndexExpanded` is set to `false`, not `index.sparse`. Is this intentional, or should it be the former instead?

Reply:
Good catch. I have a change ready to push when the Scalar tests are updated.